content/en/docs/topics/advanced.md

```diff
@@ -48,9 +48,19 @@ If you wish to use multiple post-renderers, call all of them in a script or
 together in whatever binary tool you have built. In bash, this would be as
 simple as `renderer1 | renderer2 | renderer3`.
 
-You can see an example of using `kustomize` as a post renderer
+You can see an example of using `kustomize` as a post-renderer
 [here](https://github.com/thomastaylor312/advanced-helm-demos/tree/master/post-render).
 
+#### Filenames
+Each manifest passed to the post-renderer includes a temporary annotation:
```

Comment:
It kind of sounds like each manifest is past to the post-render and if they have the annotation, the file name is not changed.

Reply:
I struggled with the wording a little bit. Do you have a suggestion that would make it clearer?

In most cases, the post-renderer can safely ignore the annotation. It will then be passed back to Helm, which uses it to restore its internal state. However, the post-renderer may modify the annotation if needed. It can also add manifests and use the annotation to indicate which file they originated from, if that information is useful.

Reply:
```suggestion
Each manifest passed to the post-renderer may include a temporary annotation:
```